styles/common.css

```diff
@@ -1,5 +1,15 @@
 @import 'colors.css';
 @import 'underlines/underlines.css';
+@import 'container/container.css';
+@import 'container/title.css';
+
+@import 'header/header.css';
```

Comment:
Styles for sections are not common styles. If you want to have a main css file for imports - then you can create `main.css` and import everything there.